yarn-project/pxe/src/pxe.ts

```diff
@@ -1072,13 +1078,13 @@ export class PXE {
    * @param recipients - The addresses that decrypted the logs.
    * @returns - The deserialized events.
```

Comment:
```suggestion
   * @returns - The events.
```

We now don't perform decoding here so that was a lie.

i think it would be fine to altogether drop the return value description here as it's clear.